### DISCUSSION TITLE
Escaping single quotes and double quotes in expressions

### DISCUSSION THREAD

**Discussion by thivi** [community/community#40449] [Actions] [answered]
### Select Topic Area

Question

### Body

I am trying to create a GitHub workflow that would automatically send me a message using a webhook whenever a new issue is created. So, my request body looks something like this:
```
{
    "title":  '${{ github.event.issue.title }}',
    "body":  '${{ github.event.issue.body }}'
}
```

However, this becomes problematic whenever the issue title/body has a `'`.  I can encapsulate the expressions within double quotes, but that would become problematic if the content contains a double quote.  Is there a way to escape the content of the issue title/body?

**Comment by airtower-luna** [comment, marked as answer]
The Actions syntax (`${{ ... }}`) will be literally substituted before whatever parses your script ever sees it. The safe way is to pass the value as an environment variable or parameter instead, so it's not parsed by the interpreter. Exactly how to do it depends on the context, the fragment you posted could fit into multiple programming languages. :wink:
